src/designer/designer_packages/elsa-workflows-designer/src/modules/workflow-definitions/components/workflow-properties/properties.tsx

```diff
@@ -179,6 +180,18 @@ export class WorkflowDefinitionPropertiesEditor {
             <option value="true" selected={this.workflowDefinition.usableAsActivity}>Yes</option>
           </select>
         </FormEntry>
+      },
+      {
+        name: 'autoUpdateConsumingWorkflows',
+        order: 0,
+        content: () => 
+          <CheckboxFormEntry fieldId="UpdateConsumingWorkflows" label="Auto-update consuming workflows" hint="When you publish new version, all the consuming workflows will be updated to point the new version of this workflow.">
```

Comment:
`"When you publish **a** new version, all **of** the consuming workflows will be updated to point **to** the new version of this workflow."`